mbuild/conversion.py

```diff
@@ -97,19 +97,26 @@ def load(
             **kwargs,
         )
     # Second check if we are loading SMILES strings
-    elif smiles:
+    elif smiles and (backend == "rdkit" or backend is None):
```

Comment:
Should we use `.lower()` on `backend` here? Incase someones enters `backend="RDkit"` or something like that?

Reply:
Yes that's an improvement!